.github/workflows/go.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
```

Comment:
Does this have a stable version of Ubuntu, I think better to go with stable rather than bleeding edge ?